sqlglot/expressions.py

```diff
@@ -3472,15 +3472,21 @@ class Command(Expression):
 
 
 class Transaction(Expression):
-    arg_types = {"this": False, "modes": False}
+    arg_types = {"this": False, "modes": False, "mark": False}
 
 
 class Commit(Expression):
-    arg_types = {"chain": False}
+    arg_types = {
+        "chain": False,
+        "this": False,
+        "transaction": False,
+        "durability": False,
+        "modes": False,
```

Comment:
Doesn't seem like `modes` is used anywhere. Let's delete it if it isn't needed.

Reply:
modes is used by Presto

Reply:
presto.py: 348
```python
def transaction_sql(self, expression: exp.Transaction) -> str:
            modes = expression.args.get("modes")
            modes = f" {', '.join(modes)}" if modes else ""
            return f"START TRANSACTION{modes}"
```

Reply:
You're right about the Commit(), fixing.